src/oas3/transformers/content.ts

```diff
@@ -103,3 +104,11 @@ const transformExamples = (source: MediaTypeObject | HeaderObject) => (key: stri
     key,
   };
 };
+
+const transformSchema = (schema: SchemaObject): JSONSchema4 | JSONSchema6 | JSONSchema7 => {
```

Comment:
Clone deep is not necessary, suggest something like this maybe?

```ts
const transformSchema = (schema: SchemaObject): JSONSchema4 | JSONSchema6 | JSONSchema7 => {
   if (schema && schema.nullable) {
     return {
       ...schema,
       type: [schema.type, 'null']
     }
   }

   return schema;
};
```

Reply:
Done!